### PR TITLE
[SimpleTimeParameterization] Throw if device velocities are not bounded

### PR DESCRIPTION
**Pull request by florent-lamiraux**
add also a few assertions.

### CODE REVIEW

**Comment by jmirabel** [comment]
I did the same this morning: 9108a8c

**Comment by florent-lamiraux** [comment]
I do not see this commit in the repository.
Should I close this pull request ?

**Comment by jmirabel** [comment]
No, that was a mere remark. Sorry for the misunderstanding.

File: include/hpp/core/time-parameterization/polynomial.hh [hunk 1]
```diff
@@ -102,6 +108,7 @@ namespace hpp {
               tn *= t;
               res += a[i] * tn;
             }
+            assert (!std::isnan(res));
```
Comment (jmirabel) [inline include/hpp/core/time-parameterization/polynomial.hh:111]: This is C++ >= 11. Can you enclose it within:
```C++
#if __cplusplus__ >= 201103L
#endif
```

Comment (jmirabel) [reply include/hpp/core/time-parameterization/polynomial.hh:111]: Alternatively, you can use `res == res` which fails only if res is a nan.

Comment (florent-lamiraux) [reply include/hpp/core/time-parameterization/polynomial.hh:111]: I did it and force pushed.

**Pull request merged**